Syntaxes/SCSS.sublime-syntax

```diff
@@ -262,6 +269,195 @@ contexts:
+
+  image-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  image-set-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  conic-gradient-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  radial-gradient-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  circle-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  inset-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  polygon-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  rect-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  cubic-bezier-timing-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  step-timing-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-angles-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-angles-scalars-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-lengths-percentage-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-scalars-percentage-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-scalars-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-angle-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-length-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-length-percentage-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  transform-function-scalar-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  url-function-arguments-list-body:
+    - meta_content_scope: meta.function-call.arguments.css meta.group.css
+    - include: function-arguments-prototype
+    - include: scss-unqualified-functions
+    - include: quoted-urls
+    - include: unquoted-urls
+
+  url-regexp-function-arguments-list-body:
+    - meta_content_scope: meta.function-call.arguments.css meta.group.css
+    - include: function-arguments-prototype
+    - include: scss-unqualified-functions
+    - include: quoted-strings
+
+  calc-function-arguments-list-body:
+    - meta_append: true
+    - include: scss-identifiers-or-functions
+
+  toggle-function-arguments-list-body:
```

Comment:
Oof, I guess we’ll just have to keep good track of updates to the CSS package and coordinate updates. It’s not a problem per se, but it’s what I was worried about when getting “entangled” with it. This would be easier if this weren’t a separate package but instead added to the default packages.

Reply:
In the long term, CSS needs to parse function more lazily to reduce the number of contexts required. The current amount of function related contexts is insane anyway.

Reply:
Yeah, it’s so super specific about what’s allowed in specific contexts, that can’t be maintainable 😅